storage/gcsemu/server.go

```diff
@@ -27,8 +27,9 @@ func NewServer(laddr string, opts Options) (*Server, error) {
 	gcsEmu := NewGcsEmu(opts)
 	mux := http.NewServeMux()
 	mux.HandleFunc("/", gcsEmu.Handler)
+	mux.HandleFunc("/batch/storage/v1", gcsEmu.BatchHandler)
```

Comment:
Ultimately we'll need to rebase this once #17 lands.  No big deal, just making note.

Reply:
rebased